src/access.rs

```diff
@@ -133,20 +133,17 @@ where
 		for txid in registered_txs {
 			if let Some(tx_status) = client.get_tx_status(&txid).await? {
```

Comment:
Much of the loop body looks repeated below. Would it makes sense to refactor into a method returning an `Option` such that the `push` and `continue` could be conditioned at the call sites?

Reply:
Now doing this in https://github.com/lightningdevkit/ldk-lite/pull/9/commits/273617adb8ba8d22bfe0fdba8d12309da50d8601.